ml-testing-toolkit/Chart.yaml

```diff
@@ -36,11 +36,6 @@ dependencies:
     version: 11.1.0
     repository: "file://./chart-connection-manager-backend"
     condition: connection-manager-backend.enabled
-  - name: percona-xtradb-cluster
-    version: 1.0.8
-    repository: https://charts.helm.sh/stable
-    alias: mysql
-    condition: mysql.enabled
   - name: mongodb
```

Comment:
Same applies here to MongoDB (i.e. another instance or schema).

Reply:
again restored ttl/Chart.yaml